packages/codegen-doc/src/constants.ts

```diff
@@ -1,4 +1,18 @@
 export const Doc = {
+  MUTATION_TYPES: [
+    "update",
+    "settingsUpdate",
```

Comment:
do we need to have this specific types? Can't it be covered by `update`? 

The same question applies to `resourceArchive` and `archive`.

Reply:
This is legacy I have moved around. Let me have a look, I agree with you we shouldn't need this.